README.md

```diff
@@ -255,8 +255,7 @@ make cluster-down
 
 # Releasing
 
-Steps to create a new release:
-
-1. Test operator, make sure it deploys all components, exposes failures in NetworkAddonsConfig.Status field as well as progressing status of components and "Ready".
-2. Open a new PR with two commits. The first of them adding new released manifests, the second bumping versions in `Makefile`. To make this easier, use `./hack/release.sh <previous version> <released version> <future version> <origin remote> <fork remote>`
-3. Once the PR is merged, tag its **first** commit with proper version name `x.y.z`. This can be done through [GitHub UI](https://github.com/kubevirt/cluster-network-addons-operator/releases/new).
+1. Checkout a public branch
```

Comment:
where should this branch be created from? is it from master? 
Also, can you be more specific about what you mean by public branch? do you mean create a new branch and pushing to the remote repository in order for it to be public?

Reply:
It is created from the code you are sitting on at the moment. That makes it pretty flexible. I hope to add an automation executed through GitHub as a follow-up.